templates/taxi/index.html

```diff
@@ -0,0 +1,9 @@
+{% extends 'base.html' %}
+
+{% block content %}
+ <ul>
+   <li>Number of cars: {{ num_drivers }}</li>
+   <li>Number of drivers: {{ num_manufacturers }}</li>
+   <li>Number of manufacturers: {{ num_cars }}</li>
+ </ul>
+{% endblock %}

```

Comment:
Add a break line to all html files

Reply:
Could you provide a link to some document, what is the break line you mean?
I'm not sure that we had something similar in the course. Thanks